monit/templates/monitrc

```diff
@@ -0,0 +1,321 @@
+###############################################################################
```

Comment:
so you want me to remove this altogether and everything the bit in monit.sh that creates it? I believe we should have monit setup in such way that it has some basic functionality straigth out of the box so that our users will see that it works at least.